src/components/Headquarter.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'styled-components';
+import imgBg from "../images/BG-4.png";
+import msGold from "../images/logo-ms-gold.png"
+const Wrapper = styled.section`
+  background: url(${imgBg}) fixed left top no-repeat;
+  background-size: cover;
+  position: relative;
+`
+const AreaInner = styled.div`
+  background: rgba(0, 0, 0, 0.5);
+  padding: 150px 0 80px;
+`
+const Title = styled.h2`
+  font-size: 2.5em;
+  margin: 0 0 30px;
+  text-align: center;
+  color: #fff;
+`
+const Subtitle = styled.h3`
+  text-align: center;
+  color: #fff;
+  padding-bottom: 50px;
+  font-size: 1.5em;
+`
+
+const Basearea = styled.div`
+  ul {
+    list-style: none;
+    padding: 0;
+    margin-left: -50px;
+    text-align: center;
+    overflow: hidden;
+
+    li {
+      display: inline-block;
+      margin: 0 0 50px 50px;
+      border: 1px solid #fff;
+      padding: 10px 40px;
+      color: #fff;
+      font-size: 1.3em;
+    }
+  }
+`
+const Msgold = styled.li`
```

Comment:
This style will display Microsoft Gold Icon. I think it you say `MSGoldIcon` instead.